src/assets/scss/input-password.scss

```diff
@@ -7,22 +7,21 @@
 .password-label {
     font-size: 16px;
     font-weight: bold;
-    box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.1);
     margin-bottom: 12px;
     margin-left: 15px;
 }
 
 // Input Password => placeholder => 'Senha'
-.password-input, .password-input:focus {
+.password-input, .password-input:focus, .password-input:disabled {
```

Comment:
Maybe it's more readable if you try something like:
.password {
    &-input, &-input:focus, &-input:disabled {
         // styles
    }
}
That's take anything with follows password-input:'something'.

Reply:
Did it, thanks for the tips!